<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageWindow</class>
 <widget class="QMainWindow" name="ImageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>760</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>760</height>
   </size>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1" rowspan="2" colspan="6">
     <widget class="VoltageTemperatureCoordinate" name="widget_3" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="2" colspan="3">
     <widget class="VoltageCoordinate" name="widget_4" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>332</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="4" rowspan="2" colspan="3">
     <widget class="TemperatureCoordinate" name="widget_5" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>330</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="4">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>310</width>
        <height>1000</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QTextEdit" name="textEdit">
         <property name="styleSheet">
          <string notr="true">font: 75 italic 9pt &quot;Arial&quot;;</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:9pt; font-weight:72; font-style:italic;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="Savedata"/>
    <addaction name="Saveimage"/>
    <addaction name="Saveresults"/>
    <addaction name="Singout"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="Parameter"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>运行</string>
    </property>
    <addaction name="Start"/>
    <addaction name="Suspend"/>
    <addaction name="Continue"/>
    <addaction name="Calculation"/>
    <addaction name="Getresult"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="Savedata">
   <property name="text">
    <string>保存（数据）</string>
   </property>
  </action>
  <action name="Saveimage">
   <property name="text">
    <string>保存（图像）</string>
   </property>
  </action>
  <action name="Saveresults">
   <property name="text">
    <string>保存（结果）</string>
   </property>
  </action>
  <action name="Singout">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actions_4">
   <property name="text">
    <string>s</string>
   </property>
  </action>
  <action name="Parameter">
   <property name="text">
    <string>参数设置</string>
   </property>
  </action>
  <action name="actions_6">
   <property name="text">
    <string>s</string>
   </property>
  </action>
  <action name="actions_7">
   <property name="text">
    <string>s</string>
   </property>
  </action>
  <action name="actions_8">
   <property name="text">
    <string>s</string>
   </property>
  </action>
  <action name="Start">
   <property name="text">
    <string>开始运行</string>
   </property>
  </action>
  <action name="Suspend">
   <property name="text">
    <string>暂停接收</string>
   </property>
  </action>
  <action name="Continue">
   <property name="text">
    <string>继续接收</string>
   </property>
  </action>
  <action name="Calculation">
   <property name="text">
    <string>拟合计算</string>
   </property>
  </action>
  <action name="Getresult">
   <property name="text">
    <string>得到结果</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VoltageTemperatureCoordinate</class>
   <extends>QWidget</extends>
   <header>voltagetemperaturecoordinate.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VoltageCoordinate</class>
   <extends>QWidget</extends>
   <header>voltagecoordinate.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TemperatureCoordinate</class>
   <extends>QWidget</extends>
   <header location="global">temperaturecoordinate.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
